packages/wrangler/src/user/user.ts

```diff
@@ -1014,7 +1014,7 @@ export async function login(
 			}
 		});
 
-		server.listen(8976);
+		server.listen(8976, "localhost");
```

Comment:
`localhost` can resolve to more than one IP address - e.g. 127.0.0.1 and [::1].
Should we, perhaps, just use 127.0.0.1 as a guaranteed single host IP?

Reply:
I purposefully picked `localhost` to ensure the possibility of dual IPv4/IPv6 support.

As you can see, clients are asked to connect to `localhost`:
https://github.com/cloudflare/workers-sdk/blob/7697c18e4874ab5ae8c68d6eb8beb23f396ecc54/packages/wrangler/src/user/user.ts#L383

Due to happy eyeballs, browsers will try to connect to `::1` and `127.0.0.1`, and accept whatever that works. So even if Node.js only listens on one of the addresses, that is fine.

As for the port availability checks, no client actually tries to connect to them. So in that case, it does not matter whether IPv4 or IPv6 support is in use.

In general I would try to use hostnames instead of IP addresses to make it easier to support both IPv4 and IPv6 at the same time.

Reply:
Cool. I can live with this explanation. Thanks!